marimbabot_audio/src/onset_detection.py

```diff
@@ -127,7 +127,8 @@ def init_detection_config(self):
 			confidence threshold for note classification(crepe)
 		"""
 		# For onset detection
-		self.confidence_threshold = 0.7  # the threshold for note classification
+		self.confidence_threshold = rospy.get_param("~confidence_threshold", 0.7)
```

Comment:
I would not include a default, as it might cause confusions when somebody changes that.

Reply:
done